spinalcordtoolbox/deepseg/models.py

```diff
@@ -126,7 +127,33 @@
         "description": "Lumbar SC segmentation on T2w contrast with 3D UNet",
         "contrasts": ["t2"],
         "default": False,
-    }
+    },
+    "model_seg_sc_contrast_agnostic_softseg_nnunet": {
+        "url": [
+            "https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/releases/download/v2.0/model_2023-09-18.zip"
+        ],
+        "description": "Spinal cord segmentation that is agnostic to contrast using NNUnet",
+        "contrasts": ["any"],
+        "default": False,
+    },
+    "model_seg_sci_multiclass_sc_lesion_nnunet": {
+        "url": [
+            # TODO: Replace with official release (*not* from joshuacwnewton)
+            "https://github.com/joshuacwnewton/model_seg_sci/releases/download/r20230814/model-sci-multisite_r20230814.zip"
+        ],
```

Comment:
Note to self: This URL will need to be replaced once an official release on the main repo has been made.

Reply:
Note from @naga-karthik:

> However, for the SCI model, I noted that it is downloading an old version of the model (as it is the only one available as a release. This should be updated when we make a new release with the latest model!

See: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4345#issuecomment-1902737199

Reply:
Updated!

Reply:
yay! 🎉